src/Codeception/Lib/GroupManager.php

```diff
@@ -169,7 +169,7 @@ public function groupsForTest(\PHPUnit\Framework\Test $test)
                     && mb_strtolower($filename . ':' . $test->getMetadata()->getFeature()) === mb_strtolower($testPattern)) {
                     $groups[] = $group;
                 }
-                if ($test instanceof \PHPUnit\Framework\TestSuite\DataProvider) {
+                if ($test instanceof \PHPUnit\Framework\DataProviderTestSuite) {
                     $firstTest = $test->testAt(0);
```

Comment:
@DavertMik This code is so dead. testAt method was removed in PHPUnit 9.3.0 - https://github.com/sebastianbergmann/phpunit/commit/7d948a75c6cc66059873386124291fd890f9ee2a#diff-ce07bbf9be7fe14e3102ab699c8a2d6c2a995bc23f8f93871fb4f3dde098dfaa